cmd/secrets-provider/main.go

```diff
@@ -120,29 +120,26 @@ func processAnnotations(ctx context.Context, tracer trace.Tracer) error {
 
 func secretRetriever(ctx context.Context,
 	tracer trace.Tracer) (*conjur.SecretRetriever, error) {
-
-	// Create a trace Span
-	spanCtx, span := tracer.Start(ctx, "Create Conjur secret retriever")
+	// Gather authenticator config
+	_, span := tracer.Start(ctx, "Gather authenticator config")
 	defer span.End()
 
-	// Gather K8s authenticator config
-	authnConfig, err := setupAuthnConfig(spanCtx, tracer)
+	authnConfig, err := authnConfigProvider.NewConfigFromCustomEnv(ioutil.ReadFile, customEnv)
 	if err != nil {
-		log.Error(err.Error())
 		span.RecordErrorAndSetStatus(err)
+		log.Error(messages.CSPFK008E)
```

Comment:
is "CSPFK008E" the right error you want to use?

Reply:
CSPFK008E Failed to instantiate authenticator configuration